src/pages/map.tsx

```diff
@@ -130,6 +138,29 @@ const MapPage = ({ schedule }) => {
     lokasi: item
   }));
 
+  const setInitialMapBound = (mapBox: MapboxGl.Map) => {
+    // eslint-disable-next-line @typescript-eslint/no-floating-promises
+    window.navigator.permissions.query({ name: 'geolocation' }).then(data => {
```

Comment:
you'll need to handle missing `permissions` object in unsupported browsers (safari)